.vscode/settings.json

```diff
@@ -0,0 +1,34 @@
+{
+  "editor.tabCompletion": "on",
+  "editor.tabSize": 2,
+  "files.eol": "\n",
```

Comment:
VS Code respects the `.editorconfig`. Declaring the settings here again won't hurt, but it is redundant and a single source might be preferable for these settings.